package.json

```diff
@@ -51,11 +51,13 @@
     "typescript": "^3.7.2"
   },
   "dependencies": {
+    "@types/ip": "^1.1.0",
     "assert": "^2.0.0",
     "base64url": "^3.0.1",
     "bcrypto": "^4.2.8",
     "bigint-buffer": "^1.1.5",
     "dgram": "^1.0.1",
+    "ip": "^1.1.5",
```

Comment:
pretty sure this doesn't work in browser, as I remember this uses some nodejs native api ("os" maybe)